.git-blame-ignore-revs

```diff
@@ -9,3 +9,6 @@
 
 # Add pyupgrade to pre-commit
 e113e37de1ec687c68337d777f3629251b35ab28
+
+Run pre-commit autoupdate and run all hooks
```

Comment:
This should be prefixed by `#` as a comment, right?

Reply:
Oops yes

Reply:
Fixed in https://github.com/PyGithub/PyGithub/pull/2342